.vscode/launch.json

```diff
@@ -1,7 +1,4 @@
 {
-    // Use IntelliSense to learn about possible attributes.
-    // Hover to view descriptions of existing attributes.
-    // For more information, visit: https://go.microsoft.com/fwlink/?linkid=830387
     "version": "0.2.0",
     "configurations": [
```

Comment:
The file is itself removed.

- Do a ``git pull upstream master``
- Create a new branch
- Copy your changes from the other branch
- Push this new branch to github
- Raise a PR

As far as the PR goes, it works fine!

Reply:
I made a new PR on it as suggested by following the above steps. Kindly review that